[2.13] flapdoodle too old for ubuntu 22.04

### Describe the bug

We see a CI failure on the 22.3 image (using quarkus 2.13):

```
Caused by: java.lang.IllegalArgumentException: could not resolve package for V4_2:Platform{operatingSystem=Linux, architecture=X86_64, distribution=Ubuntu}
--------------
(os=Windows)
  (os=Windows and bitSize=B64) and (3.4.9-3.4.24 or 3.4.0-3.4.7 or 3.2.0-3.2.22 or 3.0.0-3.0.15 or 2.6.0-2.6.12)
    url=/win32/mongodb-win32-x86_64-{version}.zip (ZIP)
  (os=Windows and bitSize=B64) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9)
    url=/windows/mongodb-windows-x86_64-{version}.zip (ZIP)
  (os=Windows and bitSize=B64) and (4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.0-4.2.3)
    url=/win32/mongodb-win32-x86_64-2012plus-{version}.zip (ZIP)
  (os=Windows and bitSize=B64) and (4.0.0-4.0.27 or 3.6.0-3.6.23 or 3.4.9-3.4.24 or 3.4.0-3.4.7 or 3.2.0-3.2.22 or 3.0.0-3.0.15)
    url=/win32/mongodb-win32-x86_64-2008plus-ssl-{version}.zip (ZIP)
  (os=Windows and bitSize=B64) and (3.4.9-3.4.24 or 3.4.0-3.4.7 or 3.2.0-3.2.22 or 3.0.0-3.0.15 or 2.6.0-2.6.12)
    url=/win32/mongodb-win32-x86_64-2008plus-{version}.zip (ZIP)
  (os=Windows and bitSize=B64) and (3.3.1-3.3.1 or 3.5.5-3.5.5)
    url=/win32/mongodb-win32-x86_64-{version}.zip (ZIP)
  (os=Windows and bitSize=B32) and (3.2.0-3.2.22 or 3.0.0-3.0.15 or 2.6.0-2.6.12)
    url=/win32/mongodb-win32-i386-{version}.zip (ZIP)
  (os=Windows and bitSize=B32) and (3.3.1-3.3.1 or 3.5.5-3.5.5)
    url=/win32/mongodb-win32-i386-{version}.zip (ZIP)
  (os=Windows)
    fail
(os=OS_X)
  (os=OS_X and bitSize=B64) and (4.0.0-4.0.27 or 3.6.0-3.6.23 or 3.4.9-3.4.24 or 3.4.0-3.4.7 or 3.2.0-3.2.22 or 3.0.4-3.0.15)
    url=/osx/mongodb-osx-ssl-x86_64-{version}.tgz (TGZ)
  (os=OS_X and bitSize=B64) and (3.5.5-3.5.5 or 3.4.9-3.4.24 or 3.4.0-3.4.7 or 3.3.1-3.3.1 or 3.2.0-3.2.22 or 3.0.0-3.0.15 or 2.6.0-2.6.12)
    url=/osx/mongodb-osx-x86_64-{version}.tgz (TGZ)
  (os=OS_X and bitSize=B64) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9 or 4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.0-4.2.3)
    url=/osx/mongodb-macos-x86_64-{version}.tgz (TGZ)
  (os=OS_X)
    fail
(os=Linux)
  (os=Linux and (version is any of Ubuntu_18_04, Ubuntu_18_10, Ubuntu_19_04, Ubuntu_19_10, Ubuntu_20_04, Ubuntu_20_10))
    (os=Linux and bitSize=B64 and (version is any of Ubuntu_20_04, Ubuntu_20_10)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9)
      url=/linux/mongodb-linux-aarch64-ubuntu2004-{version}.tgz (TGZ)
    (os=Linux and bitSize=B64 and (version is any of Ubuntu_20_04, Ubuntu_20_10)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9)
      url=/linux/mongodb-linux-x86_64-ubuntu2004-{version}.tgz (TGZ)
    (os=Linux and bitSize=B64 and (version is any of Ubuntu_18_04, Ubuntu_18_10, Ubuntu_19_04, Ubuntu_19_10, Ubuntu_20_04, Ubuntu_20_10)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9 or 4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.0-4.2.3)
      url=/linux/mongodb-linux-aarch64-ubuntu1804-{version}.tgz (TGZ)
    (os=Linux and bitSize=B64 and (version is any of Ubuntu_18_04, Ubuntu_18_10, Ubuntu_19_04, Ubuntu_19_10, Ubuntu_20_04, Ubuntu_20_10)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9 or 4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.0-4.2.3 or 4.0.1-4.0.27 or 3.6.20-3.6.23)
      url=/linux/mongodb-linux-x86_64-ubuntu1804-{version}.tgz (TGZ)
  (os=Linux and (version is any of LINUX_MINT_19_0, LINUX_MINT_19_1, LINUX_MINT_19_2, LINUX_MINT_19_3, LINUX_MINT_20_0, LINUX_MINT_20_1, LINUX_MINT_20_2, LINUX_MINT_20_3))
    use Ubuntu_18_04 for LINUX_MINT_19_0, LINUX_MINT_19_1, LINUX_MINT_19_2, LINUX_MINT_19_3 and Ubuntu_20_04 for LINUX_MINT_20_0, LINUX_MINT_20_1, LINUX_MINT_20_2, LINUX_MINT_20_3
  (os=Linux and (version is any of DEBIAN_10, DEBIAN_9))
    (os=Linux and bitSize=B64 and (version is any of DEBIAN_9)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9 or 4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.0-4.2.3 or 4.0.0-4.0.27 or 3.6.5-3.6.23)
      url=/linux/mongodb-linux-x86_64-debian92-{version}.tgz (TGZ)
    (os=Linux and bitSize=B64 and (version is any of DEBIAN_10)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9 or 4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.1-4.2.3)
      url=/linux/mongodb-linux-x86_64-debian10-{version}.tgz (TGZ)
  (os=Linux and (version is any of CentOS_6, CentOS_7, CentOS_8, Redhat_6, Redhat_7, Redhat_8, Oracle_6, Oracle_7, Oracle_8))
    (os=Linux and bitSize=B64 and (version is any of CentOS_6, Redhat_6, Oracle_6)) and (4.4.11-4.4.11 or 4.4.0-4.4.9 or 4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.0-4.2.3 or 4.0.0-4.0.27 or 3.6.0-3.6.23 or 3.4.9-3.4.24 or 3.4.0-3.4.7 or 3.2.0-3.2.22 or 3.0.0-3.0.15)
      url=/linux/mongodb-linux-x86_64-rhel62-{version}.tgz (TGZ)
    (os=Linux and bitSize=B64 and (version is any of CentOS_7, Redhat_7, Oracle_7)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9 or 4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.0-4.2.3 or 4.0.0-4.0.27 or 3.6.0-3.6.23 or 3.4.9-3.4.24 or 3.4.0-3.4.7 or 3.2.0-3.2.22 or 3.0.0-3.0.15)
      url=/linux/mongodb-linux-x86_64-rhel70-{version}.tgz (TGZ)
    (os=Linux and bitSize=B64 and (version is any of CentOS_8, Redhat_8, Oracle_8)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9 or 4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.1-4.2.3 or 4.0.14-4.0.27 or 3.6.17-3.6.23 or 3.4.24-3.4.24)
      url=/linux/mongodb-linux-x86_64-rhel80-{version}.tgz (TGZ)
    (os=Linux and bitSize=B64 and (version is any of CentOS_8, Redhat_8, Oracle_8)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.4-4.4.9)
      url=/linux/mongodb-linux-aarch64-rhel82-{version}.tgz (TGZ)
  (os=Linux and (version is any of AmazonLinux, AmazonLinux2))
    (os=Linux and bitSize=B64 and (version is any of AmazonLinux2)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.4-4.4.9 or 4.2.18-4.2.18 or 4.2.13-4.2.16)
      url=/linux/mongodb-linux-aarch64-amazon2-{version}.tgz (TGZ)
    (os=Linux and bitSize=B64 and (version is any of AmazonLinux2)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9 or 4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.0-4.2.3 or 4.0.0-4.0.27 or 3.6.22-3.6.23)
      url=/linux/mongodb-linux-x86_64-amazon2-{version}.tgz (TGZ)
    (os=Linux and bitSize=B64 and (version is any of AmazonLinux)) and (5.0.5-5.0.5 or 5.0.0-5.0.2 or 4.4.11-4.4.11 or 4.4.0-4.4.9 or 4.2.18-4.2.18 or 4.2.5-4.2.16 or 4.2.0-4.2.3 or 4.0.0-4.0.27 or 3.6.0-3.6.23 or 3.4.9-3.4.24 or 3.4.0-3.4.7 or 3.2.0-3.2.22 or 3.0.0-3.0.15)
      url=/linux/mongodb-linux-x86_64-amazon-{version}.tgz (TGZ)
  (os=Linux and bitSize=B64) and (4.0.0-4.0.26 or 3.6.0-3.6.22 or 3.4.9-3.4.23 or 3.4.0-3.4.7 or 3.2.0-3.2.21 or 3.0.0-3.0.14 or 2.6.0-2.6.12)
    url=/linux/mongodb-linux-x86_64-{version}.tgz (TGZ)
  (os=Linux and bitSize=B32) and (3.2.0-3.2.21 or 3.0.0-3.0.14 or 2.6.0-2.6.12)
    url=/linux/mongodb-linux-i686-{version}.tgz (TGZ)
  (os=Linux and bitSize=B64) and (3.3.1-3.3.1 or 3.5.5-3.5.5)
    url=/linux/mongodb-linux-x86_64-{version}.tgz (TGZ)
  (os=Linux and bitSize=B32) and (3.3.1-3.3.1 or 3.5.5-3.5.5)
    url=/linux/mongodb-linux-i686-{version}.tgz (TGZ)
  (os=Linux)
    fallback to Ubuntu_20_04
(os=Solaris)
  (os=Solaris and bitSize=B64) and (3.4.0-3.4.5 or 3.2.0-3.2.14 or 3.0.0-3.0.15 or 2.6.0-2.6.12)
    url=/sunos5/mongodb-sunos5-x86_64-{version}.tgz (TGZ)
  (os=Solaris and bitSize=B64) and (3.3.1-3.3.1 or 3.5.5-3.5.5)
    url=/sunos5/mongodb-sunos5-x86_64-{version}.tgz (TGZ)
  (os=Solaris)
    fail

	at de.flapdoodle.embed.mongo.packageresolver.PlatformPackageResolver.lambda$packageFor$0(PlatformPackageResolver.java:66)
	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
	at de.flapdoodle.embed.mongo.packageresolver.PlatformPackageResolver.packageFor(PlatformPackageResolver.java:60)
	at de.flapdoodle.embed.process.store.ArtifactStore.filesToExtract(ArtifactStore.java:93)
	at de.flapdoodle.embed.process.store.ExtractedArtifactStore.extractFileSet(ExtractedArtifactStore.java:84)
	at de.flapdoodle.embed.process.runtime.Starter.prepare(Starter.java:59)
	at de.flapdoodle.embed.process.runtime.Starter.prepare(Starter.java:52)
	at io.quarkus.test.mongodb.MongoTestResource.doGetExecutable(MongoTestResource.java:108)
	at io.quarkus.test.mongodb.MongoTestResource.getMongodExecutable(MongoTestResource.java:96)
	at io.quarkus.test.mongodb.MongoTestResource.start(MongoTestResource.java:71)
	at io.quarkus.test.common.TestResourceManager$TestResourceEntryRunnable.run(TestResourceManager.java:471)
	... 4 more
```

See: https://github.com/graalvm/mandrel/actions/runs/4836206494/jobs/8619430443#step:11:526

### Expected behavior

Flapdoodle recent enough for it to recognize Ubuntu 22.04

### Actual behavior

Test fail due to outdated version. See:
https://github.com/quarkusio/quarkus/blob/2.13/bom/application/pom.xml#L171

### How to Reproduce?

Run mongodb client integration test on Quarkus 2.13 + 22.3.

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

Latest main branch is on newer flapdoodle, which doesn't seem to have that issue.

--- a/.github/workflows/weekly.yml
+++ b/.github/workflows/weekly.yml
@@ -67,7 +67,7 @@ jobs:
     name: "Q 2.13 M 22.3 image"
     uses: graalvm/mandrel/.github/workflows/base.yml@default
     with:
-      quarkus-version: "2.13.4.Final"
+      quarkus-version: "2.13.7.Final"
       builder-image: "quay.io/quarkus/ubi-quarkus-mandrel-builder-image:22.3-java17"
   q-main-mandrel-22_3-quayio:
     name: "Q main M 22.3 image"
@@ -75,46 +75,3 @@ jobs:
     with:
       quarkus-version: "main"
       builder-image: "quay.io/quarkus/ubi-quarkus-mandrel-builder-image:22.3-java17"
-  ####
-  # Test Quarkus with supported Mandrel versions using the Quay.io images
-  ####
-  q-2_7-mandrel-21_3-quayio:
-    name: "Q 2.7 M 21.3 image"
-    uses: graalvm/mandrel/.github/workflows/base.yml@default
-    with:
-      quarkus-version: "2.7.5.Final"
-      builder-image: "quay.io/quarkus/ubi-quarkus-mandrel:21.3-java11"
-  ####
-  # Test Quarkus main with supported Mandrel versions using the release archives
-  ####
-  #q-main-mandrel-20_3-release
-  #  name: "Q main M 20.3 release"
-  #  uses: graalvm/mandrel/.github/workflows/base.yml@default
-  #  with:
-  #    quarkus-version: "main"
-  #     version: "mandrel-20.3.1.2-Final"
-  #     built-type: "mandrel-release"
-  #q-main-mandrel-21_0-release
-  #  name: "Q main M 21.0 release"
-  #  uses: graalvm/mandrel/.github/workflows/base.yml@default
-  #  with:
-  #    quarkus-version: "main"
-  #     version: "mandrel-21.0.0.0-Final"
-  #     built-type: "mandrel-release"
-  ####
-  # Test Quarkus main with supported GraalVM versions using the release archives
-  ####
-  #q-main-graal-20_3-release
-  #  name: "Q main G 20.3 release"
-  #  uses: graalvm/mandrel/.github/workflows/base.yml@default
-  #  with:
-  #    quarkus-version: "main"
-  #     version: "vm-20.3.2"
-  #     built-type: "graal-release"
-  #q-main-graal-21_0-release
-  #  name: "Q main G 21.0 release"
-  #  uses: graalvm/mandrel/.github/workflows/base.yml@default
-  #  with:
-  #    quarkus-version: "main"
-  #     version: "vm-21.1.0"
-  #     built-type: "graal-release"

